ONSdigital/ssdc-rm-support-tool
# 2950 update sample

# Motivation and Context
Adds the ability to update non-sensitive sample data via the support tool 

# What has changed
- New button on the case details screen to allow sample data to be updated

# How to test?
- Run this branch with the common-entity-model branch
- Go into the case details for a case and click the 'Update Sample' button
- Enter new details for a field. 
- After saving, you should see the UPDATE_SAMPLE event  and the relevant field should be updated

# Links
- https://trello.com/c/rGcj9FUl
- https://github.com/ONSdigital/ssdc-rm-ddl/pull/59
